.eslintrc.js

```diff
@@ -4,6 +4,11 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 6,
     sourceType: "module",
+    project: [
```

Comment:
There are more details here: https://eslint.org/docs/latest/user-guide/configuring/configuration-files .  Had to look this up because I've never seen it before. I _think_ the last two array entries may be redundant, since the root `.tsconfig.json` references the client/server specific tsconfig

Reply:
This was required to to adding a rule from the typescript-estlint plugin. Without adding the project setting, the following rule would be seen:

You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.

It the client and server tsconfigs are removed, there will be numerous errors to this effect:

```
0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: server/src/sas/SyntaxProvider.ts.
The file must be included in at least one of the projects provided
```